### PR TITLE
Update js template to support font awesome

### PR DESCRIPTION
**Pull request by corylanou**
The latest version of font-awesome requires the js file to also be present for a glyph to work.  To see this work, add this code to any page template:

```html
<h3>
<a href=#" target="_blank"><i class="fab fa-twitter-square share" aria-hidden="true"></i></a>
</h3>
```

Without this JS file, the glyph will not render.

### CODE REVIEW

File: genny/assets/webpack/templates/assets/js/application.js.tmpl [hunk 1]
```diff
@@ -1,5 +1,6 @@
 require("expose-loader?$!expose-loader?jQuery!jquery");
 require("bootstrap/dist/js/bootstrap.bundle.js");
+require("@fortawesome/fontawesome-free/js/all.js");
```
Comment (timraymond) [inline genny/assets/webpack/templates/assets/js/application.js.tmpl:3]: Should this be `@fontawesome` instead?

Comment (corylanou) [reply genny/assets/webpack/templates/assets/js/application.js.tmpl:3]: no for whatever reason they renamed it...  I did a double take the first time as well.  Such is the life of front end development righ?

**Comment by markbates** [comment]
@corylanou can you please rebase against development? thanks.

File: genny/assets/webpack/templates/assets/js/application.js.tmpl (hunk 1 above)
Comment (paganotoni) [reply genny/assets/webpack/templates/assets/js/application.js.tmpl:3]: I confirm what @corylanou says:
https://www.npmjs.com/package/@fortawesome/fontawesome-free

**Pull request merged**